e2e/sectioned-form.e2e-spec.ts

```diff
@@ -1,67 +1,159 @@
 import {
   expect,
-  SkyHostBrowser
+  SkyHostBrowser,
+  SkyVisualThemeSelector
 } from '@skyux-sdk/e2e';
 
 import {
   element,
   by
 } from 'protractor';
 
+import {
```

Comment:
Nitpicky - but could we move this import statement to be directly below the other `@skyux-sdk/e2e` import? That way they are together when looking at the imports?

Reply:
yep. Rearranged these in recent commit. https://github.com/blackbaud/skyux-tabs/commit/f23bd3b82e0f0bcaa9face16e77ee5d8720a483d